config.sample.yaml

```
@@ -113,6 +113,10 @@ ircService:
         delayMinMs: 3600000 # 1h
         # Default: 7200000, = 2h
         delayMaxMs: 7200000 # 2h
+        # Should the bridge drop any joins that happen during the debounce period. If your
+        # bridge suffers from extensive downtime when debouncing, this might help smooth
+        # things over.
```

What does smooth things over mean?
It would be good to highlight the advantages and downfalls of this, similar to what you wrote in the PR description.
```suggestion
        # bridge suffers from extensive downtime when debouncing, this might keep the bridge
        # operable by not adding new IRC users as Matrix room members. A new users will
        # get added when they post a message.
```